htdocs/catalog/catalog.php

```diff
@@ -127,7 +127,7 @@ function delete_catalog_data()
 		<div id='tdel_msg' class='clean-orange' style='display:none;'>
 			<?php echo LangUtil::$generalTerms['MSG_DELETED']; ?>&nbsp;&nbsp;<a href="javascript:toggle('tdel_msg');"><?php echo LangUtil::$generalTerms['CMD_HIDE']; ?></a>
 		</div>
-		<?php $page_elems->getTestTypeTable($lab_config_id); ?>

```

Comment:
This fixes the "Test Catalog" screen - it was completely broken since `$lab_config_id` is not defined, but it is instead stored in the `$_SESSION`